KomaMRIBase/src/motion/motionlist/Motion.jl

```diff
@@ -172,5 +186,5 @@ function Base.view(m::Motion, p)
 end
 
 # Auxiliary functions
-times(m::Motion) = times(m.time)
+times(m::Motion{T}, ::Type{T}) where {T<:Real} = times(m.time)
```

Comment:
I am confused, why do you need this type T parameter?

Reply:
Anyway, I can leave the change of the plot to the other PR.